app/helpers/view_components/site_header.rb

```diff
@@ -7,20 +7,20 @@ class SiteHeader < ViewComponent
 
     def to_s
       tag.header(id: "site-header") do
-        # announcement_bar +
-        tag.div(class: "lg-container container") do
-          logo + docs_nav + contextual_section
-        end
+        announcement_bar +
+          tag.div(class: "lg-container container") do
+            logo + docs_nav + contextual_section
+          end
       end
     end
 
     private
     def announcement_bar
-      link_to(Exercism::Routes.hiring_about_path, class: "announcement-bar") do
+      link_to(Exercism::Routes.community_path, class: "announcement-bar") do
         tag.div(class: "lg-container") do
-          tag.span("We’re hiring!") +
-            tag.strong("Come and join the Exercism team") +
-            tag.span("😁")
+          tag.span("Forums, Swag, Stories, Vides...") +
+            tag.strong("Check out new Community tab!") +
```

Comment:
```suggestion
            tag.strong("Check out our new Community tab!") +
```
```suggestion
            tag.strong("Check our new Community tab!") +
```

maybe one of these sounds better?